e2e/internal/e2e/image.go

```diff
@@ -50,3 +53,28 @@ func EnsureImage(t *testing.T) {
 		t.Fatalf("Unexpected failure: %+v", err)
 	}
 }
+
+// MakeTmpDir will make a tmp dir and return a string of the path, and a full
+// path (with a container name 'test_container.sif').
+func MakeTmpDir(t *testing.T) (string, string) {
+	name, err := ioutil.TempDir("", "stest.")
+	if err != nil {
+		t.Fatalf("Failed to create temporary directory: %v", err)
+	}
+	if err := os.Chmod(name, 0777); err != nil {
+		t.Fatalf("Failed to chmod temporary directory: %v", err)
+	}
+	fullName := name
```

Comment:
why is this here? The function is creating a temporary directory. What's "test_container.sif"?

Reply:
This function did return a full container path eg. `/tmp/tests/test_container.sif`, and a tmp path eg. `/tmp/tests`. This defiantly needs to be improved.